src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -48,6 +50,37 @@ const defaultProps = {
 };
 
 function ReportActionItemCreated(props) {
+    const {windowWidth, isSmallScreenWidth} = useWindowDimensions();
+
+    const IMAGE_OFFSET_X = windowWidth / 2;
+    const IMAGE_OFFSET_Y = 75;
+    const ANIMATION_BOOST = 1.3;
+
+    // Get data from phone rotation sensor and prep other variables for animation
+    const animatedSensor = useAnimatedSensor(SensorType.GYROSCOPE);
+    const moveXoffset = useSharedValue(0);
+    const moveYoffset = useSharedValue(0);
+    const backgroundImageOffsetX = useSharedValue(-IMAGE_OFFSET_X);
```

Comment:
Does this need to be a shared value? It never changes so it seems like it could just be a constant?

Reply:
And if so it can probably be removed and we can just reference `-IMAGE_OFFSET_X` directly?